test/model/user_test.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:checks/checks.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:zulip/api/model/events.dart';
+import 'package:zulip/api/model/model.dart';
+
+import '../api/model/model_checks.dart';
+import '../example_data.dart' as eg;
+import 'test_store.dart';
+
+void main() {
+  group('userDisplayName', () {
+    test('on a known user', () {
+      final user = eg.user(fullName: 'Some User');
+      final store = eg.store()..addUser(user);
```

Comment:
nit: One more spot that needs to be awaited.

Reply:
Hmm indeed, thanks. My sweep missed that because I looked for `store\.add`, and didn't spot method cascades like these.

Looks like this is the only example in the tree, though.